scss/main.scss

```diff
@@ -1,3 +1,4 @@
 @import 'html5-bp_main';
-@import 'unhcr';
-@import 'user_location_control';
\ No newline at end of file
+@import 'user_location_control';
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,300,700);
+@import 'unhcr_new';
```

Comment:
I think you can just replace `_unhcr.scss` with your new one unless there's a reason to keep the old one?